client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -65,6 +65,12 @@ function getHoldMessages( siteSlug, translate ) {
 			description: translate(
 				'Please try again in a few minutes: you will be able to proceed once we finish setting up your security settings.'
 			),
+		},
+		EMAIL_UNVERIFIED: {
+			title: translate( 'Unconfirmed email' ),
+			description: translate(
+				'Please check your email to confirm your address.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'To update, check your email and confirm your address.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).